### ISSUE TITLE
Installation and requirements instructions missing

### ISSUE THREAD

**Issue by dpshelio** [sunpy/solarbextrapolation#9]
If we want people to use this we need to have detailed installation instructions.

**Comment by wtbarnes** [comment]
Agreed. There should be a brief description of how to install in the README and then more detailed instructions in the docs. Right now it seems that the RTD docs just have the API documentation.

**Comment by Cadair** [comment]
I am partly waiting on this: https://github.com/conda-forge/staged-recipes/pull/1581 to build a conda package for this.

**Comment by wtbarnes** [comment]
@Cadair Ah ok... How important is it to have mayavi bundled with this package? I know it is very powerful, but I've found it to be nuisance when I'm just looking to do a quick visualization.

I've got pretty nice method for visualizing the field lines using the yt streamline tracer. This is by no means a replacement for mayavi, but maybe mayavi could be an optional dependency if people want more full-featured visualizations and yt could provide a nice quick look? Just an idea...

[Relevant example](https://github.com/wtbarnes/synthesizAR/blob/master/examples/magnetic_skeleton_example.ipynb) I think I've linked to this before.

**Comment by Cadair** [comment]
We could look at doing that, especially if you fancy PRing your yt code. I guess we will just need a little re-working of the imports to make sure we only exclude functionality that relies on mayavi if it's not there.

I would still be hesitant to make a conda package with (even optionally) the current mayavi provided by anaconda as it's *really* old.

**Comment by wtbarnes** [comment]
Ok fair enough. I may take a stab at the yt stuff. Maybe some docs improvements as well. The PR you linked seems to be fairly active; fingers crossed!

**Comment by wtbarnes** [comment]
Sooooooo now that mayavi is in conda forge...can this be added to the openastronomy conda channel? I guess the tests should pass first (i.e. I should work on #11)

**Comment by Cadair** [comment]
yes, now that exists we can work out adding this to conda-forge as well

**Comment by Cadair** [comment]
we should also look at porting this to python 3

**Comment by Alex-Ian-Hamilton** [comment]
I can look into porting it, tho time is limited... But it is my baby... And I wanna be able to run it again!!! :D

**Comment by Cadair** [comment]
I am game to just move it to python 3 and drop support for 2.x which should make it much easier

**Comment by wtbarnes** [comment]
I have been using this with Python 3 for the past few months though I haven't been messing with the mayavi parts

**Comment by Cadair** [comment]
well that's encouraging!!

**Comment by wtbarnes** [comment]
I think #8 fixed all of the trivial 2-to-3 issues. There may still be some lurking about.
